lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelTest.java

```diff
@@ -1221,14 +1222,14 @@ public void onChannelStateChanged(ChannelStateChange stateChange) {
 				channel = channelName;
 				flags |= 1 << Flag.resumed.ordinal();
 			}};
-			ably.connection.connectionManager.onMessage(attachedMessage);
+			ably.connection.connectionManager.onMessage(null, attachedMessage);
```

Comment:
It was in presence PR. There was a bug of not setting transport to null on transport disconnect and then under some race conditions allowing receiving messages from dead transport. So we need to pass transport as an argument to onMessage() to detect dead transports and null is special case to bypass check. Just for tests.